docs/developer-docs/multi-chain/supported-chains.mdx

```diff
@@ -0,0 +1,118 @@
+| Polkadot  | ECDSA, EdDSA                                                           | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |
+| Cosmos    | ECDSA, EdDSA                                                           | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |
+| NEAR      | EdDSA                                                                  | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |
+| Kaspa     | ECDSA                                                                  | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |
+| Aptos     | ECDSA, EdDSA                                                           | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |
+| Monero    | EdDSA                                                                  | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |
+| Hedera    | ECDSA, EdDSA                                                           | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |
+| Filecoin  | ECDSA                                                                  | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |
+| Stacks    | ECDSA                                                                  | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |
+| Sui       | ECDSA, EdDSA                                                           | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |
+| Arweave   | [Not yet supported](https://github.com/ArweaveTeam/arweave/issues/486) | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |
+| Stellar   | EdDSA                                                                  | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |
```

Comment:
```suggestion

| Aptos     | [ECDSA, EdDSA](https://aptos.dev/en/network/blockchain/accounts)                                                           | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |    
| Arweave   | [Not yet supported](https://github.com/ArweaveTeam/arweave/issues/486) | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |
| Avalanche | [ECDSA](https://docs.avax.network/reference/standards/cryptographic-primitives)                                                                 | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |
| Bitcoin   | [ECDSA, Schnorr](/docs/current/developer-docs/multi-chain/bitcoin/using-btc/generate-addresses)                                                        | Direct                                                                                                                                         | [ckBTC](docs/developer-docs/multi-chain/chain-key-tokens/ckbtc/overview.mdx)                                                                    |
| Cardano   | [ECDSA, EdDSA, Schnorr ](https://developers.cardano.org/docs/governance/cardano-improvement-proposals/cip-0049/)                                                 | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |                                                                                                                                                 |
| Cosmos    | [ECDSA, EdDSA](https://docs.cosmos.network/v0.50/learn/beginner/accounts)                                                           | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |
| Dogecoin  | ECDSA                                                                  | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |                                                                                                                                                 |
| Ethereum  | [ECDSA](/docs/current/developer-docs/multi-chain/ethereum/using-eth/generating-addresses)                                                                  | [EVM RPC](docs/developer-docs/multi-chain/ethereum/evm-rpc/overview.mdx), [ic-web3-rs](https://github.com/horizonx-tech/ic-web3-rs)            | [ckETH](docs/developer-docs/multi-chain/chain-key-tokens/cketh/overview.mdx), ckERC20                                                           |
| Filecoin  | [ECDSA](https://spec.filecoin.io/algorithms/crypto/signatures/)                                                                  | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |
| Hedera    | [ECDSA, EdDSA](https://docs.hedera.com/hedera/core-concepts/keys-and-signatures)                                                           | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |
| Kaspa     | [ECDSA](https://github.com/kaspanet/go-secp256k1/blob/master/ecdsa_privkey.go)                                                                  | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |
| Monero    | [EdDSA](https://getmonero.dev/cryptography/asymmetric/edwards25519.html)                                                                  | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |
| NEAR      | EdDSA                                                                  | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |
| Polkadot  | [ECDSA, EdDSA](https://wiki.polkadot.network/docs/learn-cryptography)                                                           | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |
| Solana    | [EdDSA](https://solana.com/docs/core/accounts#account)                                                                  | [Solana RPC](https://github.com/mfactory-lab/ic-solana) <span style={{ display: 'inline-block', verticalAlign: 'super' }}><AlphaChip /></span> | [gSOL](https://galactic-bridge-tzbjl.ondigitalocean.app/) <span style={{ display: 'inline-block', verticalAlign: 'super' }}><BetaChip /></span> |
| Stacks    | [ECDSA](https://github.com/stacks-network/keylib-py)                                                                  | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |
| Stellar   | [EdDSA](https://developers.stellar.org/docs/smart-contracts/smart-wallets#secp256r1)                                                                  | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |
| Sui       | [ECDSA, EdDSA](https://blog.sui.io/wallet-cryptography-specifications/)                                                           | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |
| XRP       | [ECDSA, EdDSA](https://xrpl.org/docs/concepts/accounts/cryptographic-keys/#signing-algorithms)                                                           | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |                                                                                                                                                 |
| Toncoin   | EdDSA                                                                  | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |                                                                                                                                                 |
| TRON      | [ECDSA](https://github.com/tronprotocol/tips/issues/120)                                                                  | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |                                                                                                                                                 |
```

Reply:
Reorganized by alphabetical order and added links to each chain's documentation that specifies its supported signatures

Reply:
I'm not sure if this is a good idea as links may change, and there's not a huge value in having these links available.